backend/main.py

```diff
@@ -10,6 +12,50 @@
     env.reset(sys.argv[1])
     visualiser = Visualiser(env.realm)
 
+    #Temporary constants
+    car_size = 2
+    
     while True:
-        env.step({})
+        actions = {}
+        #Work out actions
+        for actor in env.realm.actors.values():
+            if type(actor) is Depot:
+                for t in actor._storage:
+                    truck = actor._storage[t]
+                    if truck.done == False: #Truck is waiting to be released
+                        next_road = env.realm.edges[truck.next_container_id]
+                        if type(next_road) is Road:
```

Comment:
Maybe this should be made an `assert` statement instead, since the next container after a Depot *should* always be a road